test/Tests/Writers/Docbook.hs

```diff
@@ -32,9 +32,11 @@ which is in turn shorthand for
 -}
 
 infix 4 =:
-(=:) :: (ToString a, ToPandoc a)
-     => String -> (a, String) -> TestTree
+infix 4 =::
+(=:), (=::) :: (ToString a, ToPandoc a)
+            => String -> (a, String) -> TestTree
 (=:) = test docbook
+(=::) = test docbook5
```

Comment:
I do not really like this operator, it looks like assigning type.
Ideally we would have `(=:5)` but Haskell does not seem to allow numbers in operators.

Reply:
Agree, it's a bit confusing. I think we could define `testdb5` or `db5` and set ``infix 4 `db5` ``. Or we could do fancy stuff with unicode symbols like `⚄` and `⚃`, but, uuuh, maybe not.

Reply:
Or maybe just simple `test docbook5 "test name" $`

Reply:
:+1:

Reply:
Done.